modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -129,8 +129,13 @@ exports.signin = function (req, res, next) {
  * Signout
  */
 exports.signout = function (req, res) {
+  var redirectURL = 'https://logon.gov.bc.ca/clp-cgi/logoff.cgi?returl=' + req.headers.referer;
```

Comment:
This logic requires validation in test to confirm it doesn't break the existing functionality.
It should however just redirect to the homepage if the currently logged in user is "local", instead of trying to go through siteminder (since it is unnecessary).
This should make the logout functionality work correctly when in local/development mode.